app.go

```diff
@@ -42,14 +41,13 @@ func New(appID string, config *Config) (*App, error) {
 		client: client,
 	}
 
-	app.jwksCache = jwk.NewCache(context.Background())
-	if err := app.jwksCache.Register(fmt.Sprintf(jwksUrl, appID)); err != nil {
+	cache := jwk.NewCache(context.Background())
+	if err := cache.Register(fmt.Sprintf(jwksUrl, appID)); err != nil {
 		return nil, err
 	}
 
-	if err := app.refreshJWKSCache(); err != nil {
-		return nil, err
-	}
+	app.jwksCacheSet = jwk.NewCachedSet(cache, fmt.Sprintf(jwksUrl, appID))
+	var _ jwk.Set = app.jwksCacheSet
```

Comment:
this isn't necessary. it was only included in the [example](https://github.com/lestrrat-go/jwx/blob/536b643c661a886300246d2f334d4d836e61f497/docs/04-jwk.md?plain=1#L710) to show that it fulfills the `jwk.Set` interface

Reply:
Removed this.